### ISSUE TITLE
Retry download artwork step

### ISSUE THREAD

**Issue by mcauser** [get-iplayer/get_iplayer#191]
Occasionally I have downloaded a show and it fails at the `INFO: MP4 tagging M4A file` step.

```
$ get_iplayer --type=radio --modes=best --thumb --thumbsize 11 --pid=b064y7qc
...
INFO: Recorded /Users/foo/get_iplayer/BBC_Radio_1s_Essential_Mix_Essential_Mix_Masters_-_Essential_Mix_Masters_Jose_Padilla_b064y7qc_default.m4a
INFO: Downloaded Thumbnail to '/Users/foo/get_iplayer/BBC_Radio_1s_Essential_Mix_Essential_Mix_Masters_-_Essential_Mix_Masters_Jose_Padilla_b064y7qc_default.jpg'
Wide character in print at /opt/boxen/homebrew/bin/get_iplayer line 5310.
INFO: MP4 tagging M4A file
Image '/Users/foo/get_iplayer/BBC_Radio_1s_Essential_Mix_Essential_Mix_Masters_-_Essential_Mix_Masters_Jose_Padilla_b064y7qc_default.jpg' could not be loaded.
INFO: Command exit code 1 (raw code = 256)
WARNING: Failed to tag M4A file
```

Retry with `--force` and it works, but has to download the entire show again.
I can download the artwork only with `--thumbnail-only` and then run AtomicParsley to insert the artwork manually.

It would be nice if the script could retry to download artwork once, or add a setting for number of retries.

Has anyone else experienced something similar?

brew get_iplayer v2.94, on osx 10.10.5

**Comment by dinkypumpkin** [comment]
Please post feature requests at: https://squarepenguin.co.uk/forums/

**Comment by dinkypumpkin** [comment]
Closed by df2dfa0af2d163dbd94e352e304b4a1386c8f2c3
